scripts/quantize.py

```diff
@@ -100,19 +97,8 @@ def main(args_in: Optional[List[str]] = None) -> None:
     args = parser.parse_args(args_in)
 
     model_name = model_maps.get(args.model_name, args.model_name)
-    if is_win():
-        path = Path(args.build_dir, "./Bin/Release/quant_{}.exe".format(model_name))
-    else:
-        if args.one_click_run == "True":
-            import neural_speed
-            package_path = os.path.dirname(neural_speed.__file__)
-            path = Path(package_path, "./quant_{}".format(model_name))
-        else:
-            path = Path(args.build_dir, "./bin/quant_{}".format(model_name))
-    if not path.exists():
-        print(path)
-        print("Please build graph first or select the correct model name.")
-        sys.exit(1)
+    package_path = os.path.dirname(neural_speed.__file__)
+    path = Path(package_path, "./quant_{}".format(model_name))
```

Comment:
no need distinguish the windows and linux ?

Reply:
No need. Same code works for both Linux and Windows, you can test it by yourself :)